examples/RDF_to_NX.ipynb

```diff
@@ -0,0 +1,460 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example Graph Visualization with Cytoscape\n",
+    "\n",
+    "Very basic for now. Just to illustrate the LPG.\n",
+    "\n",
+    "> Note: Cytoscape appears to cast tuples to list for the visualization."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "\n",
+    "import ipycytoscape\n",
+    "import ipywidgets as W"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Here we will build a cytoscape representation of the RDF graph to show the reduces complexity\n",
+    "from rdflib.extras.external_graph_libs import rdflib_to_networkx_multidigraph\n",
+    "\n",
+    "cyto_from_rdf = ipycytoscape.CytoscapeWidget()\n",
+    "cyto_from_rdf.graph.add_graph_from_networkx(\n",
+    "    rdflib_to_networkx_multidigraph(rdf_graph),\n",
+    "    multiple_edges=True,\n",
+    "    directed=True,\n",
+    ")\n",
+    "for node in cyto_from_rdf.graph.nodes:\n",
+    "    # deal with inability to handle colons\n",
+    "    node.data[\"_label\"] = str(node.data.get(\"id\", None))\n",
+    "\n",
+    "cyto_from_rdf.set_layout(animate=False)\n",
+    "cyto_from_rdf.set_style(\n",
+    "    [\n",
+    "        {\n",
+    "            \"selector\": \"node\",\n",
+    "            \"css\": {\n",
+    "                \"label\": \"data(_label)\",\n",
+    "                \"text-wrap\": \"wrap\",\n",
+    "                \"text-max-width\": \"150px\",\n",
+    "                \"text-valign\": \"center\",\n",
+    "                \"text-halign\": \"center\",\n",
+    "                \"font-size\": \"10\",\n",
+    "                \"font-family\": '\"Gill Sans\", sans-serif',\n",
+    "                \"color\": \"blue\",\n",
+    "            },\n",
+    "        },\n",
+    "        {\n",
+    "            \"selector\": \"edge.directed\",\n",
+    "            \"style\": {\n",
+    "                \"curve-style\": \"bezier\",\n",
+    "                \"target-arrow-shape\": \"triangle\",\n",
+    "            },\n",
+    "        },\n",
+    "        {\"selector\": \"edge.multiple_edges\", \"style\": {\"curve-style\": \"bezier\"}},\n",
+    "    ]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "directed = ipycytoscape.CytoscapeWidget()\n",
+    "directed.graph.add_graph_from_networkx(nx_graph, multiple_edges=True, directed=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for node in directed.graph.nodes:\n",
+    "    # deal with inability to handle colons\n",
+    "    node.data[\"_label\"] = node.data.get(\"rdfs:label\", None)\n",
+    "    node.data[\"_attrs\"] = json.dumps(\n",
```

Comment:
also thinking that maybe eventually we want to phase out the hovering _attrs showing up just because it's kind of cluttery at times... maybe we can move to the JSON tool as an attached widget for people who want to see specific attrs. Not sure though. Definitely good as is

Reply:
Absolutely. I hate that it is in there. Once we get your latest work in I want to have a widget that integrates the Cytoscape widget with the JSON viewer.